<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端.主动链接到服务器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/titli.png</normaloff>:/img/titli.png</iconset>
  </property>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineHost">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.189</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linPort">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>129</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>10005</string>
   </property>
  </widget>
  <widget class="QPushButton" name="requestPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>128</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLocal">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置IP</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstIP">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtHasSend">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽正黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(85, 0, 0);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtStatus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>接收</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="fromTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽正黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(0, 85, 0);</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="toLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>编译报文</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="toTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽正黑</family>
    </font>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'文泉驿等宽正黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'宋体'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCall1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>召唤1级数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCall2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>召唤2级数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRLK">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>召唤链路状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>S3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>480</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>64</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>错误状态:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRCU">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>复位链路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReadTI">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>读电量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXL">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>300</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>读需量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReadTime">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>读取时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>S2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(FCV无效)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(FCV无效)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtSend">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>真实发送</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtOnlyShow">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>仅显示报文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSP_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>读单点信息</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
